index.html

```diff
@@ -230,6 +230,27 @@
         <dt>
           midi
         </dt>
+	<dt>
+	  camera
+	</dt>
+	<dt>
+	  microphone
+	</dt>
+	<dt>
+	  speaker
+	</dt>
+	<dt>
+	  all-cameras
+	</dt>
+	<dt>
+	  all-microphones
+	</dt>
+	<dt>
+	  all-speakers
```

Comment:
I wonder if we could have the "all" cases expressed in the `PermissionDescriptor` instead.